.travis.rosinstall.kinetic

```diff
@@ -0,0 +1,86 @@
+- git:
+    uri: 'https://github.com/ipa320/cob_calibration_data.git'
+    local-name: cob_calibration_data
+    version: indigo_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_android.git'
+    local-name: cob_android
+    version: indigo_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_command_tools.git'
+    local-name: cob_command_tools
+    version: indigo_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_common.git'
+    local-name: cob_common
+    version: indigo_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_control.git'
+    local-name: cob_control
+    version: kinetic_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_driver.git'
+    local-name: cob_driver
+    version: indigo_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_environments.git'
+    local-name: cob_environments
+    version: indigo_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_extern.git'
+    local-name: cob_extern
+    version: indigo_dev
+- git:
+    local-name: cob_gazebo_plugins
+    uri: 'https://github.com/ipa320/cob_gazebo_plugins.git'
+    version: kinetic_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_hand.git'
+    local-name: cob-hand
+    version: indigo_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_manipulation.git'
```

Comment:
we are not starting the collision_monitor with the cob_bringup , but you are right the dependency is listed